Source/WebCore/platform/mediastream/RealtimeMediaSourceSettings.cpp

```diff
@@ -168,11 +153,11 @@ OptionSet<RealtimeMediaSourceSettings::Flag> RealtimeMediaSourceSettings::differ
 String convertEnumerationToString(VideoFacingMode enumerationValue)
```

Comment:
We can remove this one and use the JSVideoFacingMode.h version?

Reply:
I will do that when it exists.